app/clients/sim_data.py

```diff
@@ -213,45 +213,103 @@ def sim_get_books(*args, **kwargs):
 
 
 def sim_get_magazines(*args, **kwargs):
-    return [
-        {
-            "title": "issue 10",
-            "filename": "latest_magazine.pdf"
-        },
-        {
-            "title": "issue 9",
-            "filename": "latest_magazine.pdf"
-        },
-        {
-            "title": "issue 8",
-            "filename": "latest_magazine.pdf"
-        },
-        {
-            "title": "issue 7",
-            "filename": "latest_magazine.pdf"
-        },
-        {
-            "title": "issue 6",
-            "filename": "latest_magazine.pdf"
-        },
-        {
-            "title": "issue 5",
-            "filename": "latest_magazine.pdf"
-        },
-        {
-            "title": "issue 4",
-            "filename": "latest_magazine.pdf"
-        },
-        {
-            "title": "issue 3",
-            "filename": "latest_magazine.pdf"
-        },
-        {
-            "title": "issue 2",
-            "filename": "latest_magazine.pdf"
-        },
-        {
-            "title": "issue 1",
-            "filename": "latest_magazine.pdf"
-        }
-    ]
+    return [{"title": "issue 10",
+             "type": "Philosophy",
```

Comment:
As mentioned in chat, the type belongs to articles and not magazines. Also articles will be tagged with different keywords, I'll create something this weekend to demonstrate the use of tags in articles

Reply:
I see you updated `sim_get_article`, can we instead update `sim_get_articles_summary` as it would be better to use the `short_content` that it has?

Reply:
I updated the get_article as an example, feel free to update get_articles_summary as you will need to to get working